src/pick.js

```diff
@@ -9,7 +8,7 @@ var _indexOf = require('./internal/_indexOf');
  * @func
  * @memberOf R
  * @category Object
- * @sig [String] -> {String: *} -> {String: *}
+ * @sig [k] -> {k: v} -> {k: v}
```

Comment:
I simply reused the signature from `pickAll`. But I agree with you. Should I change it back to the old signature as well as changing `pickAll`?